### PR TITLE
Port InlineKeyboardButton functions

### PR DESCRIPTION
**Pull request by MrYacha**
# Description

Ported 'add', 'row' and 'insert' function from dev-2.x

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Documentation (typos, code examples or any documentation update)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: aiogram/api/types/inline_keyboard_markup.py [hunk 1]
```diff
@@ -18,5 +18,49 @@ class InlineKeyboardMarkup(MutableTelegramObject):
     Source: https://core.telegram.org/bots/api#inlinekeyboardmarkup
     """
 
-    inline_keyboard: List[List[InlineKeyboardButton]]
+    row_width: int = 3
```
Comment (JrooTJunior) [inline aiogram/api/types/inline_keyboard_markup.py:21]: Custom fields should be placed after all fields described in API docs with separating by one empty line in due to mixing the content make updating auto-generated code harder.

File: aiogram/api/types/inline_keyboard_markup.py [hunk 2]
```diff
@@ -18,5 +18,49 @@ class InlineKeyboardMarkup(MutableTelegramObject):
...
+    inline_keyboard: List[List[InlineKeyboardButton]] = []
```
Comment (JrooTJunior) [inline aiogram/api/types/inline_keyboard_markup.py:22]: Do you know something about mutability in Python?

File: aiogram/api/types/inline_keyboard_markup.py [hunk 3]
```diff
@@ -18,5 +18,49 @@ class InlineKeyboardMarkup(MutableTelegramObject):
...
     """Array of button rows, each represented by an Array of InlineKeyboardButton objects"""
+
+    def add(self, *args: InlineKeyboardButton) -> InlineKeyboardMarkup:
+        """
+        Add buttons
+        :param args:
```
Comment (JrooTJunior) [inline aiogram/api/types/inline_keyboard_markup.py:28]: Please add one empty line between description and params list

File: aiogram/api/types/inline_keyboard_markup.py [hunk 4]
```diff
@@ -18,5 +18,49 @@ class InlineKeyboardMarkup(MutableTelegramObject):
...
+        :return: self
+        :rtype: :obj:`types.InlineKeyboardMarkup`
```
Comment (JrooTJunior) [inline aiogram/api/types/inline_keyboard_markup.py:30]: `rtype` is not needed in due to used type annotations.

File: aiogram/api/types/inline_keyboard_markup.py (hunk 2 above)
Comment (evgfilim1) [reply aiogram/api/types/inline_keyboard_markup.py]: @JrooTJunior it'll work as needed anyway. [Proof/source](https://stackoverflow.com/questions/63793662/how-to-give-a-pydantic-list-field-a-default-value)

**Comment by Masynchin** [comment]
What is the point of this keyboard be both mutable and constructable? Under the nood it uses lists to store rows and buttons so why returning it back? Or why why using lists if it designed to be immutable?

**Comment by JrooTJunior** [comment]
Replaced by keyboard builder - https://docs.aiogram.dev/en/dev-3.x/utils/keyboard.html

**Pull request closed without merging**